app.py

```diff
@@ -145,5 +145,35 @@ def station_detail_page(id):
     response = requests.get(API_BASE_URL, params={
         'key': OPEN_CHARGE_MAP_KEY, 'countrycode': 'US', 'ID': id})
     data = get_results(response)
+    return render_template('details.html', data=data)
+
+
+############################# User Feedback ###############################
+
+
+@app.route('/station/detail/<string:username>')
+def users_feedbacks(username):
+    print('inside station route')
     user = User.query.all()
-    return render_template('details.html', data=data, user=user)
+    print(user)
+    comments = Comment.query.filter_by(user_name=username)
+    print(comments)
```

Comment:
In this route I am able to get the user but not getting comments. And It's not rendering my template 'stationfeedback.html'.
What's going wrong I don't understand.
Please take a look and help me out.
Thanks